src/connection/connection.go

```diff
@@ -357,7 +358,7 @@ func GetBrokerFromZookeeper(zkConn zookeeper.Connection, id, preferredListener s
 		// Check that the protocol map
 		protocol, ok := brokerDecoded.ProtocolMap[listener]
 		if !ok {
-			log.Error("Listener '%s' was not found in the protocol map")
+			log.Error("Listener '%s' was not found in the protocol map", listener)
```

Comment:
What is the type of `listener`? If it doesn't include `host:port` in the name might be good logging those as well. Otherwise this should be fine as it is :)